linetools/spectralline.py

```diff
@@ -291,8 +290,7 @@ def aodm(self, **kwargs):
             if len(lim) > 0:
                 sub = np.maximum(0.05, sig[satp[lim]]/5.)
                 nndt[satp[lim]] = np.log(1./sub)*cst
-                flg_sat = len(lim) 
-                raise ValueError('USE FLG_SAT')
+                flg_sat = True
```

Comment:
Ok great, please merge when you're ready.

On Mon, Jun 29, 2015 at 11:29 PM, J. Xavier Prochaska <
notifications@github.com> wrote:

> In linetools/spectralline.py
> <https://github.com/linetools/linetools/pull/7#discussion_r33463085>:
>
> > @@ -291,8 +290,7 @@ def aodm(self, **kwargs):
> >              if len(lim) > 0:
> >                  sub = np.maximum(0.05, sig[satp[lim]]/5.)
> >                  nndt[satp[lim]] = np.log(1./sub)*cst
> > -                flg_sat = len(lim)
> > -                raise ValueError('USE FLG_SAT')
> > +                flg_sat = True
>
> AODM no longer incoporated in this PR
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/linetools/linetools/pull/7/files#r33463085>.
>